### PR TITLE
Alerting: Change the rule yaml data to reflect Prom-based rule format

### PR DESCRIPTION
**Pull request by konrad147**
**What this PR does / why we need it**:
This PR allows users to copy the YAML rule data from the UI and use it in the `alerts.yaml` configuration files for Prometheus/Mimir

**Special notes for your reviewer**:
**BEFORE**
image

**AFTER**
image

### CODE REVIEW

**Review by soniaAguilarPeiron — APPROVED** [review]
👏 LGTM

File: public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/Query.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const Query: FC = () => {
           <InputControl
             name="expression"
             render={({ field: { ref, ...field } }) => {
+              console.log('Expr value', field.value);
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/Query.tsx:34]: This console?

Comment (konrad147) [reply public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/Query.tsx:34]: Ooops...

**Comment by konrad147** [comment]
Added YAML context notification
image

File: public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx [hunk 2]
```diff
@@ -111,6 +127,25 @@ const InspectorYamlTab: FC<YamlTabProps> = ({ onSubmit }) => {
   );
 };
 
+function YamlContentInfo() {
+  return (
+    <div>
+      The YAML content in the editor represents a single alert only. <br />
+      It does not contain the complete configuration of alert rules needed for Prometheus configuration
```
Comment (konrad147) [inline public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx:134]: @brendamuir Could you take a look at this message? We'd like to be clear that we display only a part of the necessary YAML configuration

File: public/app/features/alerting/unified/components/rule-editor/ExpressionEditor.tsx [hunk 3]
```diff
@@ -26,7 +27,7 @@ export const ExpressionEditor: FC<ExpressionEditorProps> = ({ value, onChange, d
 
   const onChangeQuery = useCallback(
     (query: DataQuery) => {
-      setQuery(query);
+      // setQuery(query);
```
Comment (peterholmberg) [inline public/app/features/alerting/unified/components/rule-editor/ExpressionEditor.tsx:30]: Should this be commented?

Comment (konrad147) [reply public/app/features/alerting/unified/components/rule-editor/ExpressionEditor.tsx:30]: No, I'll remove :)

**Review by peterholmberg — APPROVED** [review]
Looks good from my end, nice work!

File: public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx (hunk 2 above)
Comment (brendamuir) [reply public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx:134]: A single alert / alert rule? Do you wnat to state that in the first part of this sentence - like "The YAML content in the editor only displays part of the configuration required for an alert rule.(Prometheus as well - or something else?) It does not contain the complete configuration required for the Prometheus configuration of an alert rule. " does that make sense?

Comment (konrad147) [reply public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx:134]: Some customers may use this YAML editor to copy the content and then use it as the content of a Prometheus configuration file. 
What I want to emphasize is the content contains only an alert rule and if you want to use that as a config file, then you should be aware that this is only part of the configuration file

Comment (brendamuir) [reply public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx:134]: The YAML content in the editor only contains alert rule configuration. To configure Prometheus, copy the entire configuration file.

**Pull request merged**